msal/src/main/java/com/microsoft/identity/nativeauth/NativeAuthPublicClientApplication.kt

```diff
@@ -491,6 +508,14 @@ class NativeAuthPublicClientApplication(
                 )
             }
 
+            if (username.isBlank()) {
+                return@withContext SignUpError(
+                    errorType = ErrorTypes.INVALID_USERNAME,
+                    errorMessage = "Empty or blank username",
+                    correlationId = "UNSET"   //TODO use null correlation id here and in other places in this file instead of UNSET
```

Comment:
Why this TODO? Why can't be left UNSET?

Reply:
This was old TODO when `correlationId` was nullable in your PR. This has now been removed.